aptos-move/framework/aptos-framework/sources/coin.spec.move

```diff
@@ -288,13 +288,16 @@ spec aptos_framework::coin {
         decimals: u8,
         monitor_supply: bool,
     ): (BurnCapability<CoinType>, FreezeCapability<CoinType>, MintCapability<CoinType>) {
-        pragma aborts_if_is_partial;
+        //pragma aborts_if_is_partial;
```

Comment:
if the spec works, you could just remove this comment

Reply:
fixed